finance/accounting.md

```diff
@@ -1,52 +1,68 @@
 # Accounting data
 
 Accounting data describes the money that we **know** we have spent or that we **highly expect** to receive.
-There are a few kinds of accounting data we have access to:
 
-All 2i2c-specific accounting documents should be accessible on our Team Drive to anybody with a `@2i2c.org` address.
-We don't control the access conditions for the CS&S documents described below, so if you don't have access to these, ask the {role}`Executive Director`.
+## Accounting transactions airtable
 
-We split our financial and accounting data between two places:
+{term}`CS&S` provides us a monthly dataset of the last 6 months of all of all our accounting transactions.
+This is [in a Google Sheet that is automatically updated each day][gsheet].
+It includes the last 6 months of any transaction 2i2c has made (if we need more than 6 months of history, ask `fsp@2i2c.org` for a custom data dump).
 
-- [2i2c Drive Folder with our finance assets](https://drive.google.com/drive/folders/1D5NQKhPDP6zMQ8EdLcMOceTz-ek81nmX?usp=sharing) contains 2i2c-specific budgeting and modeling information. It is available to all team members.
+We have a copy of this sheet in an [**Accounting Transactions AirTable**][accounting-table][^airtable].
+Any columns that are synchronized with CS&S have a little lightning bolt (⚡) next to them.
+We use this to create [our accounting dashboard](accounting:dashboard).
+
+[^airtable]: See [](../administration/airtable.md) for how to access AirTable.
+
+[gsheet]: https://docs.google.com/spreadsheets/d/1qH5IK18z79X8cEwlwDwnlArTiYnbRVIPGqdOUdrsF0c/edit?usp=sharing
+
+[accounting-table]: https://airtable.com/appbjBTRIbgRiElkr/tblDKGQFU0iEIa5Qb/viwAdsIgMwbqKDdZ0
+
+### Update our accounting data airtable
+
+We have a semi-automated process to update the data in our accounting table.
+Here are the steps to follow:
+
+% TODO: We should automate this with a CRON job, make.com/zapier integration, etc.
+%   ref: https://github.com/2i2c-org/team-compass/issues/703
+1. **Confirm that [the Google Sheet][gsheet] has been updated** by checking the `Automatic Operations Events Log` tab.
+2. **Delete all of the records** in the [accounting table][accounting-table].
```

Comment:
I worry about this step. I can imagine myself being tired, thinking I'm following this procedure, a deleting some other AirTable that is not machine generated.

Reply:
Agreed - for now, I am not sure how to avoid it though. The way to avoid this is in this issue but I didn't want it to block this PR:

-  https://github.com/2i2c-org/team-compass/issues/703